### DISCUSSION TITLE
[glob] is not compatible with packaged types from minimatch

### DISCUSSION THREAD

**Discussion by nicojs** [DefinitelyTyped/DefinitelyTyped#64079] [Issues with a @types package]
The `minimatch` module ships its [own types since 6.0](https://github.com/isaacs/minimatch/blob/main/changelog.md#60). The types from `@types/glob` are incompatible with these new types. 

> Namespace '"minimatch/dist/cjs/index"' has no exported member 'IOptions'.

Reproduce:

```
mkdir tmp && cd tmp
npm init --yes
npm i -D typescript @types/glob minimatch
touch index.ts
npx tsc --skipLibCheck false
```

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
can you be more specific? Sample usage causing a problem would be nice

**Reply by nicojs** [reply]
I've added reproduction steps. Do they not work for you?

**Reply by peterblazejewicz** [reply]
I could misred, those setup configuration, but the problem requires TS in 'index.ts', what's that code?

**Reply by nicojs** [reply]
No code in index.ts is required. It's only necessary to enable 'lib check' (which is why you need to run with `--skipLibCheck false`). I think TypeScript needs at least 1 input file to do actual work.

**Reply by BendingBender** [reply]
see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/63967#discussioncomment-4798340

**Comment by nicojs** [comment]
I've created a PR to fix this. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64080. There is an open question there
